### DISCUSSION TITLE
trojan的一个独有特征，可能被利用识别

### DISCUSSION THREAD

**Discussion by Xpsia** [XTLS/Xray-core#1307] [General]
配置：略
无论何种配置方案都 存在，我测试了trojan+tcp+nginx 、trojan+ws+nginx 、trojan+grpc+nginx等
希望大佬们提供解决方法或修正问题！！感恩！！

如果不是tcp时(如：ws、grpc)，只需在客户端将其协议调成tcp也会出现一样的提示。
**当type为trojan时，我们只需在客户端配置IP、端口及域名后，密码有意输错，就会提示如下信息：**

此网站无法提供安全连接
[www.google.com](http://www.google.com/) 发送的响应无效。
尝试运行 Windows 网络诊断。
ERR_SSL_PROTOCOL_ERROR

**而默认情况下的提示应该如下：**

无法访问此网站
[www.google.com](http://www.google.com/) 的响应时间过长。
请试试以下办法：

检查网络连接
检查代理服务器和防火墙
运行 Windows 网络诊断
ERR_CONNECTION_TIMED_OUT

vmess系列没有此问题。

**Comment by SorenEricMent** [comment]
这不是协议特征 是客户端实现导致的。
